fem/qfunction.cpp

```diff
@@ -76,6 +90,13 @@ void QuadratureFunction::ProjectGridFunction(const GridFunction &gf)
       // Use quadrature interpolator to go from E-vector to Q-vector
       const QuadratureInterpolator *qi =
          gf_fes.GetQuadratureInterpolator(*qs_elem);
+
+      if (!qi)
+      {
+         ProjectGridFunctionFallback(gf);
```

Comment:
That warning could go here to let the user know at least in the debug mode that the implementation does not use a quadrature interpolator and is expected to be slower 🐢 .